src/crypto/constants.ts

```diff
@@ -0,0 +1,24 @@
+import { fromHexString } from '../utils';
+
+// Umbral constants
+export const SIGNATURE_LENGTH = 64;
```

Comment:
There are `serialized_size()` methods in Umbral objects, but they are currently only exposed in Python - we can do the same for WASM. Although it won't be necessary with the advent of `nucypher-core`